Marsiple/Common/Models/Comment.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Comments.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 20/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import Foundation
+
+class Comment { //TODO: - Setup properties to correspond to Json object from api
```

Comment:
`//TODO:` markers will not be detected and displayed as warnings. It `has` to be written as `// TODO:`. Also, regarding positioning of the `TODO`/`NOTE`/`FIXME` markers, lets put them above the line of code they reference, if there's room above. If not, a marker on the same line's fine.

Reply:
They are detected when I `build`.

Reply:
`TODO`s are recognised in both cases, but `MARK`s aren't recognised if there's a space between `//` and `MARK`.

Reply:
Interesting. You're right - I tried this and it seemed that it didn't show the tag within the warnings. Feel fre to ignore this comment.